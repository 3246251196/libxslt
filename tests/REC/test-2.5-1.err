compilation error: file ./test-2.5-1.xsl line 2 element stylesheet
xsl:version: only 1.0 features are supported
