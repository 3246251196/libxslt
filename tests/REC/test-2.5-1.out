<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html>
<head><title>XSLT 1.8 required</title></head>
<body><p>Sorry, this stylesheet requires XSLT 1.8.</p></body>
</html>
